albogdan/skule-vote
# Call /messages in FE and display on elections page

## Overview

- Resolves #107 
- API is called every minute


## Unit Tests Created

- Tests new hook that calls API, and the messages component that displays them


## Steps to QA

- Create an active message on the admin dashboard. You should see it on the elections page (refresh the page so you don't have to wait for a minute)
- Create an inactive message on the admin dashboard. Refresh the elections page, you shouldn't see it
